src/main/java/prefs/Preferences.java

```diff
@@ -118,6 +118,16 @@ public void addBoard(String name, List<PanelInfo> panels) {
     public void removeBoard(String name) {
         global.removeBoard(name);
     }
+
+    public void setLastDeletedIndex(String deletedBoardName){
```

Comment:
Can omit header comment if the method signature is explanatory enough (i.e. everything you are going to say in the comment is already contained in the method signature), which is usually the case for the most trivial methods only.